service/frontend/service.go

```diff
@@ -186,6 +193,8 @@ func NewConfig(
 		PersistencePerShardNamespaceMaxQPS:    dynamicconfig.DefaultPerShardNamespaceRPSMax,
 		EnablePersistencePriorityRateLimiting: dc.GetBoolProperty(dynamicconfig.FrontendEnablePersistencePriorityRateLimiting, true),
 
+		EnableDynamicRateLimiting: dynamicconfig.DisabledDynamicRateLimiting,
```

Comment:
for constants you can do `dynamicconfig.GetBoolPropertyFn(false)` (and equivalents for other types or if you need a namespace filter one)